ConscryptMode.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Conscrypt Mode
+group: User Guide
+order: 2
+redirect_from: "/activity-lifecycle/"
+---
+
+# ConscryptMode 
+
+Robolectric can currently either use Conscrypt and BouncyCastle or just BouncyCastle as the security provider. 
+In order to migrate tests over time, there is a `ConscryptMode` annotation that controls whether Conscrypt is loaded as the default security provider with BouncyCastle as backup. 
+
+### Implementation
+
+```Java
+package org.robolectric.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Documented
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.PACKAGE, ElementType.TYPE, ElementType.METHOD})
+public @interface ConscryptMode {
+
+  /**
+   * Specifies the different supported Conscrypt modes. If ConscryptMode is ON, it will install
+   * Conscrypt. If it is OFF, it won't do that but either way BouncyCastle is still installed.
+   */
+  enum Mode {
+    ON,
+
+    OFF,
+  }
+
+  Mode value();
+}
+```
+
+### ConscryptMode ON vs ConscryptMode OFF
```

Comment:
You could probably just remove this header and merge this section with the ConscryptMode section above.